interface/forms/soap/templates/general_new.html

```diff
@@ -1,52 +1,58 @@
 <html>
 <head>
```

Comment:
still really need to cement out 1 form so gonna focus comments on the soap form
(if this sounds familiar, then it is, because I copy/pasted these comments from the last code review :) )